### PR TITLE
Ivy and Diff-Hl support

### PR DESCRIPTION
**Pull request by drot**

### CODE REVIEW

File: gotham-theme.el [hunk 1]
```diff
@@ -413,6 +413,11 @@ depending on DISPLAY for keys which are either :foreground or
    (company-tooltip-mouse :foreground base7 :background base4)
    (company-tooltip-selection :foreground base7 :background base4)
 
+   ;; diff-hl
+   (diff-hl-change :background blue)
+   (diff-hl-delete-face :foreground red)
```
Comment (wasamasa) [inline gotham-theme.el]: This should be `diff-hl-delete`.

**Comment by wasamasa** [comment]
Looks like everything is covered.  I'm not quite happy with the colour choices yet though:

diff-hl:
- Orange is the general highlight colour.  Please don't use it to indicate added lines.
- Look at the existing diff/ediff faces to know what colours to pick here.

ivy:
- It's fine to use colour backgrounds to indicate matches, but I'd pick other shades, preferably the lesser saturated ones.  Try how magenta, violet, blue, cyan, green would look like.

**Comment by drot** [comment]
I switched the Diff-Hl insert face to the "default" one, also fixed the Volatile Highlights one. Ivy faces look fine to me, the author of the packages uses color backgrounds to indicate matches.

**Comment by wasamasa** [comment]
Thanks again!

**Pull request merged**